applications/colonoscopy_segmentation/CMakeLists.txt

```diff
@@ -67,8 +67,8 @@ if(BUILD_TESTING)
                    --config ${CMAKE_CURRENT_BINARY_DIR}/colonoscopy_segmentation_testing.yaml
                    --data "${HOLOHUB_DATA_DIR}/colonoscopy_segmentation"
            WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
-  set_property(TEST colonoscopy_segmentation_python_test PROPERTY ENVIRONMENT
-               "PYTHONPATH=${GXF_LIB_DIR}/../python/lib:${CMAKE_BINARY_DIR}/python/lib")
+  set_property(TEST colonoscopy_segmentation_python_test
+                PROPERTY ENVIRONMENT PYTHONPATH="")
```

Comment:
Maybe we can just remove these lines?

Reply:
Will the test use the environment PYTHONPATH if we remove this line?

Slightly better practice to ensure that the container PYTHONPATH is not affecting results, but not a real concern either way as we expect `sys.path` to already have the Holoscan Python installation